data/strings/strings.txt

```diff
@@ -29869,3 +29869,140 @@
     vi = Bản đồ
     zh-Hans = 地图
     zh-Hant = 地圖
+
+  [search_clear_button]
+    tags = android
+    en = Clear search
```

Comment:
Translations below are incorrect, they say not to "make the search box empty", but "the search box is clear for anyone to understand". To generate better ones, I used the article "the": `Clear the search`:

```
    en = Clear search
    af = Vee die soektog uit
    ar = مسح البحث
    az = Axtarışı təmizləyin
    be = Ачысціць пошук
    bg = Изчистване на търсенето
    ca = Esborra la cerca
    cs = Vymazat vyhledávání
    da = Ryd søgningen
    de = Lösche die Suche
    el = Καθαρίστε την αναζήτηση
    es = Borrar la búsqueda
    es-MX = Limpiar la búsqueda
    et = Tühjenda otsing
    eu = Garbitu bilaketa
    fa = جستجو را پاک کنید
    fi = Tyhjennä haku
    fr = Efface la recherche
    he = נקה את החיפוש
    hi = खोज साफ़ करें
    hu = Törölje a keresést
    id = Menghapus pencarian
    it = Cancella la ricerca
    ja = 検索をクリアする
    ko = 검색 지우기
    lt = Išvalyti paiešką
    mr = शोध साफ करा
    nb = Fjern søket
    nl = Wis de zoekopdracht
    pl = Wyczyść wyszukiwanie
    pt = Limpa a pesquisa
    pt-BR = Limpar a pesquisa
    ro = Ștergeți căutarea
    ru = Очистите поиск
    sk = Vymazať vyhľadávanie
    sv = Rensa sökningen
    sw = Futa utafutaji
    th = ล้างการค้นหา
    tr = Aramayı temizleyin
    uk = Очистити пошук
    vi = Xóa tìm kiếm
    zh = 清除搜索
    zh-Hans = 清除搜索
    zh-Hant = 清除搜尋
```